packages/nouns-webapp/src/components/Bid/index.tsx

```diff
@@ -162,27 +167,35 @@ const Bid: React.FC<{
       case 'None':
         setBidButtonContent({
           loading: false,
-          content: 'Place bid',
+          content: <Trans>Place bid</Trans>,
         });
         break;
       case 'Mining':
-        setBidButtonContent({ loading: true, content: '' });
+        setBidButtonContent({ loading: true, content: <></> });
         break;
       case 'Fail':
         setModal({
-          title: 'Transaction Failed',
-          message: placeBidState.errorMessage ? placeBidState.errorMessage : 'Please try again.',
+          title: <Trans>Transaction Failed</Trans>,
+          message: placeBidState.errorMessage ? (
+            placeBidState.errorMessage
+          ) : (
+            <Trans>Please try again.</Trans>
+          ),
```

Comment:
I know this was already in place, but can make these more readable by using optional chaining and or operator.

e.g.

```ts
placeBidState?.errorMessage || <Trans>Please try again.</Trans>,
```